package.json

```diff
@@ -32,7 +32,8 @@
         ]
     },
     "dependencies": {
-        "storyblok-js-client": "^4.0.3",
-        "storyblok-vue": "^1.0.5"
+        "storyblok-js-client": "git+https://github.com/mbr74/storyblok-js-client.git",
```

Comment:
Needs to be switched to release version after storyblok-js-client MR had been approved and merged